src/dev/src/app/tree-view/eager-declarative-tree/eager-declarative-tree.html

```diff
@@ -54,6 +54,44 @@ <h4>Selection</h4>
   </clr-tree-node>
 </div>
 
+<h4>Selection with Disabled nodes</h4>
```

Comment:
I would like to have a demo of this for each of the following:

* Eager declarative tree (this file)
* Eager recursive tree (missing)
* Lazy declarative tree (missing)
* Lazy recursive tree (missing)